database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
```

Comment:
Observations: The current implementation uses procedural code and lacks error handling and data validation.

Recommendations:
- **Refactor to Object-Oriented Design**: Implement a `Database` class to manage database connections and interactions.
- **Use Dependency Injection**: Pass the `Database` instance to other classes that require database access, promoting loose coupling and easier testing.